app/controllers/bookings_controller.rb

```diff
@@ -2,6 +2,10 @@ class BookingsController < ApplicationController
 
   def show
```

Comment:
@georgkloeck  - This isn't the greatest branch name. I'm seeing "all" you worked on was the show and the index. The branch name should describe the work you're doing. Your branch name is create new garden. From that branch name one would assume that you worked on the new and create actions fro gardens. A better name would have been feature/backend-garden-show

Reply:
looks like it works tho, that's great 👍